api/src/services/files.ts

```diff
@@ -96,26 +96,23 @@ export class FilesService extends ItemsService {
 				logger.warn(err);
 			}
 
-			payload.metadata = {};
-
 			try {
-				payload.metadata = await exifr.parse(buffer.content, {
-					icc: false,
+				const exifrMetadata = await exifr.parse(buffer.content, {
 					iptc: true,
 					ifd1: true,
 					interop: true,
-					translateValues: true,
-					reviveValues: true,
-					mergeOutput: false,
 				});
```

Comment:
> I opted to change mergeOutput to true. This would simplify the keys configured by FILE_METADATA_ALLOWLIST.

Yeah, I figured! However, it does mean that the metadata from files post this PR will have a different structure than the files from pre this PR, and we don't include a migration to update those to match 🙂

Reply:
Excellent point! :)